package/smt.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+￼ Fri Jun  7 12:48:08 UTC 2019 - Will Stephenson <wstephenson@suse.com>
+￼
+￼ - Version 2.0.35
+￼ - Extend Subscription name and product class fields to 256 chars (bsc#1129844)
```

Comment:
What's with the weird `￼` symbols at the start of all those lines?

Reply:
Some UTF crap I picked up by copy and pasting them out of github, apparently. Will clean.